artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -3796,19 +3796,58 @@ public void createBridge(final String name,
                             final boolean ha,
                             final String user,
                             final String password) throws Exception {
+      createBridge(name,
+              queueName,
+              forwardingAddress,
+              filterString,
+              transformerClassName,
+              retryInterval,
+              retryIntervalMultiplier,
+              initialConnectAttempts,
+              reconnectAttempts,
+              useDuplicateDetection,
+              confirmationWindowSize,
+              clientFailureCheckPeriod,
+              staticConnectorsOrDiscoveryGroup,
+              useDiscoveryGroup,
+              ha,
+              user,
+              password,
+              ActiveMQClient.DEFAULT_CALL_TIMEOUT);
+   }
+
+   @Override
+   public void createBridge(final String name,
+                            final String queueName,
+                            final String forwardingAddress,
+                            final String filterString,
+                            final String transformerClassName,
+                            final long retryInterval,
+                            final double retryIntervalMultiplier,
+                            final int initialConnectAttempts,
+                            final int reconnectAttempts,
+                            final boolean useDuplicateDetection,
+                            final int confirmationWindowSize,
+                            final long clientFailureCheckPeriod,
+                            final String staticConnectorsOrDiscoveryGroup,
+                            boolean useDiscoveryGroup,
+                            final boolean ha,
+                            final String user,
+                            final String password,
+                            final long callTimeout) throws Exception {
       if (AuditLogger.isEnabled()) {
          AuditLogger.createBridge(this.server, name, queueName, forwardingAddress, filterString,
                   transformerClassName, retryInterval, retryIntervalMultiplier, initialConnectAttempts,
                   reconnectAttempts, useDuplicateDetection, confirmationWindowSize, clientFailureCheckPeriod,
-                  staticConnectorsOrDiscoveryGroup, useDiscoveryGroup, ha, user, "****");
+                  staticConnectorsOrDiscoveryGroup, useDiscoveryGroup, ha, user, "****", callTimeout);
       }
       checkStarted();
 
       clearIO();
 
       try {
          TransformerConfiguration transformerConfiguration = transformerClassName == null ? null : new TransformerConfiguration(transformerClassName);
```

Comment:
You should check if `transformerClassName` isEmpty to work properly with hawtio2, see ARTEMIS-3072 / #3410 for further details.

Reply:
Fixed in the master branch in the mean time. :+1: